packages/carbon-graphs/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# ChangeLog
```

Comment:
does carbon have a change log?

Reply:
Since the changelogs are not mantained at this moment, I deleted them: eb5aaddfab9d60b88e33943cb47f76f3e6fc6f2c